deploy/service.yaml

```diff
@@ -0,0 +1,186 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Docs service template for thethingsindustries.com
+Metadata:
+  AWS::CloudFormation::Interface:
+    ParameterLabels:
+      CPU:
+        default: CPU limit of a container
+      DockerImage:
+        default: Docker image
+      Memory:
+        default: Memory limit of a container
+      TaskDesiredCount:
+        default: Number of container instances to run (prod only)
+Parameters:
+  CPU:
+    AllowedPattern: .+
+    Default: '256'
+    Type: String
+  DockerImage:
+    AllowedPattern: .+
```

Comment:
This can't just take anything. Perhaps something like `^([a-z0-9.-]+(:[0-9]+)?)?(/[a-z0-9_.-]+)*([a-z0-9_.-]+)(:[A-Za-z0-9_.-]+)?$` (please check)

Reply:
1. Nobody is going to understand what this regex actually does.
2. It's far more likely and dangerous that someone enters a value that is valid, but not what they meant. If someone enters `';;';';[[;[];[` as docker image, service will just fail to start and that's it.
3. People are going to manually enter this value once a year.

Reply:
1. The regex is not shown to anyone. It's only used by CloudFormation to validate the field.
2. With the regex, it's at least less likely to accidentally add spaces and similar stupid mistakes. The fact that it doesn't catch _all_ stupid mistakes doesn't mean we shouldn't try to catch _some_.
3. Is that a reason not to validate the field?

Reply:
Ah, I agree on spaces.